Plugins/JiraCommitTipPlugin/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Atlassian.SDK" version="2.4.0" targetFramework="net40" />
```

Comment:
New plugin and its third-party libraries should be added to the installer (_Product.wxs_), otherwise end user won't get them.